code/implementation/host-api/src/main/java/io/cattle/platform/host/api/HostApiProxyTokenManager.java

```diff
@@ -86,6 +86,7 @@ protected Object createInternal(String type, ApiRequest request) {
         }
 
         String apiProxyScheme = HostApiUtils.HOST_API_PROXY_SCHEME.get();
+        hostAddress = hostAddress.replace("http://", "").replace("https://", "");
```

Comment:
Looks fine to me.  I'm guessing doing a replaceFirst() is probably not necessary